protocol/messenger_communities.go

```diff
@@ -610,7 +647,22 @@ func (m *Messenger) ImportCommunity(ctx context.Context, key *ecdsa.PrivateKey)
 		return nil, err
 	}
 
-	return m.JoinCommunity(ctx, community.ID())
+	communitySettings := communities.CommunitySettings{
+		CommunityID:                  community.IDString(),
+		HistoryArchiveSupportEnabled: true,
+	}
+	err = m.communitiesManager.SaveCommunitySettings(communitySettings)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
We actually don't need to do this here, because `JoinCommunity` will take care of this.